code/game/objects/structures/inflatable.dm

```diff
@@ -0,0 +1,316 @@
+/obj/item/inflatable
+	name = "inflatable wall"
+	desc = "A folded membrane which rapidly expands into a large cubical shape on activation."
+	icon = 'icons/obj/inflatable.dmi'
+	icon_state = "folded_wall"
+	w_class = 3
+
+/obj/item/inflatable/attack_self(mob/user)
+	playsound(loc, 'sound/items/zip.ogg', 75, 1)
+	user << "<span class='notice'>You inflate [src].</span>"
+	var/obj/structure/inflatable/R = new /obj/structure/inflatable(user.loc)
```

Comment:
Oh cool
Okay so it spawns it inside the locker, when you open the locker it pops out.